start_jupyter_cm/windows.py

```diff
@@ -60,17 +72,26 @@ def add_jupyter_here():
         os.path.dirname(__file__), 'icons'))
     logos = {'qtconsole': os.path.join(logo_path, 'jupyter-qtconsole.ico'),
              'notebook': os.path.join(logo_path, 'jupyter.ico')}
+
+    # Here I need to create a script for pythonw
+    scr_fw = open(os.path.join(PYTHON_PATH, 'Scripts',
```

Comment:
Why do you create the `sjc-jupyter-qtconsole-script.py` to start the qtconsole instead of using the identical one shipped the qtconsole?